spec/factories/user_factory.rb

```diff
@@ -1,7 +1,42 @@
+# == Schema Information
+#
+# Table name: user
+#
+#  id                  :integer          not null, primary key
+#  login               :string(255)
+#  email               :string(255)
+#  crypted_password    :string(255)
+#  password_salt       :string(255)
+#  created_at          :datetime
+#  updated_at          :datetime
+#  activated_at        :datetime
+#  remember            :boolean          default(FALSE), not null
+#  homepage            :string(255)
+#  profile             :text
+#  profile_html        :text
+#  single_access_token :string(255)      default(""), not null
+#  role_name           :string(40)
+#  perishable_token    :string(40)
+#  persistence_token   :string(128)      default(""), not null
+#  login_count         :integer          default(0), not null
+#  last_request_at     :datetime
+#  last_login_at       :datetime
+#  current_login_at    :datetime
+#  last_login_ip       :string(255)
+#  current_login_ip    :string(255)
+#  tos                 :boolean
+#
+# Indexes
+#
+#  index_user_on_last_request_at    (last_request_at)
+#  index_user_on_login              (login)
+#  index_user_on_persistence_token  (persistence_token)
+#
+
 FactoryGirl.define do
   factory :user do
     email { Faker::Internet.email }
-    login { Faker::Name.first_name }
+    login { Faker::Internet.user_name(3..100) }
```

Comment:
We should convert this and also email to use factorygirl sequences -- it's rare to get duplicates but it does happen, i..e 

```FactoryGirl.define do
  factory :user do
    sequence(:email) { |n| "email#{n}@example.com" }
  end
end```